ui/server.tsx

```diff
@@ -7,6 +7,9 @@ import { App } from 'ui/App'
 import { Config, ConfigProvider } from 'ui/components/ConfigProvider'
 import { Document } from 'ui/Document'
 import { initApollo } from 'ui/lib/initApollo'
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
?

Reply:
Due to how we set the tags var in server eslint doesn't like the fact we dont init it in this file

Reply:
Setting a var from another module like this is pretty fragile. Better to export a function `resetTagIds` that modifies the var.